BudgetSquirrel.Business/Tracking/Transaction.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using BudgetSquirrel.Business.BudgetPlanning;
+
+namespace BudgetSquirrel.Business.Tracking
+{
+  public class Transaction
+  {
+    public Guid Id { get; private set; }
+    public string VendorName { get; private set; }
+    public string Description { get; private set; }
+    public decimal Amount { get; private set; }
+    public DateTime DateOfTransaction { get; private set; }
+    public string CheckNumber { get; private set; }
+    public string Notes { get; private set; }
+    public Budget Budget { get; private set; }
```

Comment:
Should we consider making some of these options. Just thinking some users might not want to put a Note or CheckNumber.